plugins/mac-capture/mac-sck-audio-capture.m

```diff
@@ -68,8 +68,7 @@ static bool init_audio_screen_stream(struct screen_capture *sc)
     sc->stream_properties = [[SCStreamConfiguration alloc] init];
     os_sem_wait(sc->shareable_content_available);
 
-    SCDisplay * (^get_target_display)(void) = ^SCDisplay *
-    {
+    SCDisplayRef (^get_target_display)(void) = ^SCDisplayRef {
```

Comment:
Is this change correct? Or is this a manual fix?

Reply:
Manual fix because `clang-format` 17.0.3 doesn't seem to play nice with pointer types in ObjC blocks. Using `<Type>Ref` for pointer types is an established CoreFoundation pattern though.